standard/interfaces.md

```diff
@@ -436,138 +426,64 @@ A virtual method with implementation declared in an interface may be overridden
->
-> <!-- Example: {template:"standalone-lib-without-using", name:"InterfaceMethods5", expectedErrors:["CS0535"]} -->
-> ```csharp
-> interface IF
-> {
->     void M();
-> }
-> 
-> abstract class F : IF { } // error: 'F' does not implement 'IF.M'
-> ```
->
-> *end example*
-
-It is an error if a property `P` has a most specific `get` accessor in one type `T1`, and a most specific `set` accessor in another type `T2`.
+> This is useful in derived interfaces where the implementation of a method is inappropriate and a more appropriate implementation should be provided by implementing classes. *end example*
 
 ### 18.4.3 Interface properties
 
 This clause augments the description of properties in classes [§15.7](classes.md#157-properties) for properties declared in interfaces.
 
-Interface properties are declared using *property_declaration*s ([§15.7.1](classes.md#1571-general)).
+Interface properties are declared using *property_declaration*s ([§15.7.1](classes.md#1571-general)) with the following additional rules:
 
-*property_modifier* shall not include `override`.
+- *property_modifier* shall not include `override`.
+- An explicit interface member implementation shall not contain an *accessor_modifier* (§15.7.3).
+- A derived interface may implement an abstract interface property declared in a base interface.
 
-An explicit interface member implementation shall not contain an *accessor_modifier* (§15.7.3).
+  > *Note*: As an interface cannot contain instance fields, an interface property cannot be an instance auto-property, as that would require the declaration of implicit hidden instance fields. *end note*
 
-A *property_declaration* that has a *block* as an *accessor_body* is a default implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract. An instance *property_declaration* that has no default implementation is always considered abstract; it is *never* considered to be an automatically implemented property ([§15.7.4](classes.md#1574-automatically-implemented-properties)).
-
-A derived interface mey implement an abstract interface property declared in a base interface.
-
-> *Note*: As an interface cannot contain instance fields, an interface property cannot be an instance auto-property, as that would require the declaration of implicit hidden instance fields. *end note*
-
-The type of an interface property shall be output-safe if there is a get accessor, and shall be input-safe if there is a set accessor.
+- The type of an interface property shall be output-safe if there is a get accessor, and shall be input-safe if there is a set accessor.
+- A *property_declaration* that has a *block* as an *accessor_body* is an implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract. An instance *property_declaration* that has no implementation is always considered abstract; it is *never* considered to be an automatically implemented property ([§15.7.4](classes.md#1574-automatically-implemented-properties)).
```

Comment:
What about expression-bodied properties: `int Prop => 42;`
And does this cover expression-bodied accessors? `int Prop { get => 42; }`?